src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -80,6 +80,8 @@ def register_options(cls, register):
                   'create a new chroot, which will be much slower, but more correct, as the '
                   'isolation verifies that all dependencies are correctly declared.')
     register('--junit-xml-dir', metavar='<DIR>',
+             removal_version='1.5.0.dev0',
+             removal_hint='Unused. We always use JUnit XML results files.',
```

Comment:
> Does "we" here mean "Twitter"?

yep. to distribute work, we do the equivalent of a `./pants changed --dependees=transitive | xargs -L1 -P800 ./pants test` across a distributed worker pool - so this provides a convenient way to dump and gather the output for summarization.